### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by veggiemonk**
Adding the chat link where relevant (thanks @gesellix for the suggestion)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Please note that this project is released with a [Contributor Code of Conduct](h
 
 `awesome-docker` is a hand-crafted list for high-quality information about Docker and its resources. It should be related or compatible with Docker. If it's just an image built on top of Docker, the project possibly belongs to other [awesome lists](https://github.com/sindresorhus/awesome). You can check the [awesome-selfhosted list](https://github.com/Kickball/awesome-selfhosted) or the [awesome-sysadmin list](https://github.com/n1trux/awesome-sysadmin) as well.
 If it's a **tutorial or a blog post**, they get outdated really quickly so we don't really put them on the list but if it is on a very advanced and/or specific topic, we will consider it!
-If something is awesome, share it (pull request or issue or chat), let us know why and we will help you!
+If something is awesome, share it (pull request or issue or [chat](https://gitter.im/veggiemonk/awesome-docker)), let us know why and we will help you!
```
Comment (devhkr) [inline CONTRIBUTING.md]: :+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ But usually, we are pretty relaxed people, so just come and say hi, we'll figure
 - **Adding to the list:** Submit a pull request or open an issue
 - **Removing from the list:** Submit a pull request or open an issue
 - Changing something else: Submit a pull request or open an issue
-- Don't know what to do: Open an issue or join our chat, let us know what's going on.
+- Don't know what to do: Open an issue or join our [chat](https://gitter.im/veggiemonk/awesome-docker), let us know what's going on.
 
 **join the chat:**
```
Comment (devhkr) [inline CONTRIBUTING.md]: Hi @veggiemonk 

Redundancy could be avoided. Chat link is right here.

Comment (devhkr) [inline CONTRIBUTING.md]: Just got an idea: If we leave the chat link here, we could also put a link for opening an issue. What do you think?

Open an [issue](https://github.com/veggiemonk/awesome-docker/issues/new) or join our [chat](https://gitter.im/veggiemonk/awesome-docker)

File: CONTRIBUTING.md (hunk 1 above)
Comment (devhkr) [inline CONTRIBUTING.md]: Same idea here:

[pull request](https://github.com/veggiemonk/awesome-docker/blob/master/CONTRIBUTING.md#rules-for-pull-request) or [issue](https://github.com/veggiemonk/awesome-docker/issues/new) or [chat](https://gitter.im/veggiemonk/awesome-docker)

**Comment by veggiemonk** [comment]
@devhkr updated :D

**Comment by devhkr** [comment]
LGTM :smiley:

**Pull request merged**